src/serve.rs

```diff
@@ -113,9 +117,12 @@ impl<'r> Responder<'r> for RangeResponder<File> {
 		let vec_range = match Range::from_str(range_header) {
 			Ok(Range::Bytes(v)) => v,
 			_ => {
-				warn!("Ignoring range header that could not be parse {:?}, file length is {:?}", range_header, file_length);
+				warn!(
+					"Ignoring range header that could not be parse {:?}, file length is {:?}",
```

Comment:
This was incorrect before but can you fix the typo in the error message here (`parse` instead of `parsed`) while we're here.

Reply:
Fixed.